src/classes/ProfileResults.ts

```diff
@@ -62,15 +47,15 @@ module.exports = class ProfileResults {
   /**
    * Get median memory usage.
    */
-  medianMemoryUsage () {
-    return this.data.mem_list.sort((a, b) => a > b)[Math.round(this.data.mem_list.length / 2)];
+  public medianMemoryUsage () {
+    return this.data.mem_list.sort((a, b) => a - b)[Math.round(this.data.mem_list.length / 2)];
   }
 
   /**
    * Get most frequently occuring value in memory usage.
    */
-  modeMemoryUsage () {
-    const freqMap = {};
+  public modeMemoryUsage () {
+    const freqMap: any = {};
```

Comment:
I think this should have the type `Record<string, unknown>` if I am remembering properly. I believe that this isn't erroring because eslint is not setup correctly.